src/scheduler.h

```diff
@@ -79,4 +81,26 @@ class CScheduler
     bool shouldStop() { return stopRequested || (stopWhenEmpty && taskQueue.empty()); }
 };
 
+/**
+ * Class used by CScheduler clients which may schedule multiple jobs
+ * which are required to be run serially. Does not require such jobs
+ * to be executed on the same thread, but no two jobs will be executed
+ * at the same time.
+ */
+class SingleThreadedSchedulerClient {
+private:
+    CScheduler *m_pscheduler;
+
+    CCriticalSection m_cs_callbacksPending;
+    std::list<std::function<void (void)>> m_callbacksPending;
+    bool m_fCallbacksRunning = false;
```

Comment:
Would you mind changing this to lowercase-only variable names (e.g. `m_cs_callbacks_pending`, `m_callbacks_pending`, `m_callbacks_running`)?